### PR TITLE
Cleaned up tests, fixed mistake

### PR DESCRIPTION
**Pull request by meyerweb**
Two things needed to be fixed.  First, I realized that reftests don’t need descriptive text explaining what they should be, the reftest does that.  Second, I realized I had mistakenly pushed an earlier, incorrect version of the tests, leading to incorrect reftest results.  I’ve changed my processes to prevent such mistakes in the future.  My apologies.  Still learning!

### CODE REVIEW

File: css/css-flexbox/flexbox_justifycontent-left-001-ref.html [hunk 1]
```diff
@@ -9,11 +9,9 @@
   width: 40em;
   display: flex;
```
Comment (mrego) [inline css/css-flexbox/flexbox_justifycontent-left-001-ref.html:10]: Ideally we should avoid using flexbox in the reference, so it's a different codepath.

File: css/css-flexbox/flexbox_justifycontent-right-001-ref.html [hunk 2]
```diff
@@ -9,11 +9,9 @@
   width: 40em;
   display: flex;
   flex-direction: row;
-  justify-content: right;
```
Comment (mrego) [inline css/css-flexbox/flexbox_justifycontent-right-001-ref.html:12]: I'm not sure I'm getting this properly. So despite you set `justify-content: right`, the things are on the left of the flex container. Is that the expected behavior?

**Review by mrego — COMMENTED** [review]
I'm not sure if I'm getting how these tests should work exactly.

It looks like `justify-content` `left` vs `right` is not having any effect.

**Comment by meyerweb** [comment]
Okay, I restored the descriptive text with more clarity, and simplified the codepath used for reference renderings to only use normal-flow box properties.  I think it’s ready now, but let me know if not!

**Review by mrego — APPROVED** [review]
LGTM.

Really nice you avoided flexbox on the references, thanks!

**Pull request merged**